### ISSUE TITLE
Discuss the react-native integration plan

### ISSUE THREAD

**Issue by hellais** [ooni/probe#1058]
We agreed we would be discussing how to move forward the react-native integration.

Relevant to this discussion is also the fact that airbnb stopped using it: https://medium.com/airbnb-engineering/sunsetting-react-native-1868ba28e30a. 

We should research prior work to and come up with a set of questions we should answer in order fully evaluate if it's a good idea to move forward with the plan.

**Comment by lorenzoPrimi** [comment]
@sarathms @hellais can this be closed?

**Comment by hellais** [comment]
> @sarathms @hellais can this be closed?

Yes I would say we close it.

For context, during yesterdays weekly frontend meeting call it was discussed if and how we should go ahead with the react-native integration plan.

We concluded that since we don't have a lot of upcoming UX/UI work that needs to be done, the cost of integrating react-native is not justified.

Moreover the original goals of the react-native integration was to enable people like @sarathms and myself to contribute to the mobile app, but in reality we are both already quite overloaded with other react projects (Explorer, Probe Desktop, Run, etc.), so adding also this to our plate is not ideal.

In conclusion we are going to table the react-native integration plan and just continue with a truly native app for the foreseable future.

In the event this does change, we should open a new issue to track that.
